src/Sarif.FunctionalTests/SpecExamples/Comprehensive.sarif

```diff
@@ -78,6 +78,9 @@
       },
       "results": [
         {
+		  "analysisTarget": {
+			"uri": "file:///home/buildAgent/src/collections/list.cpp"
+		  },
```

Comment:
Yes! (Nit: indentation) This (list.cpp _vs_ list.h) is the canonical example for this feature.